Glyssen/Utilities/CustomColor.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Drawing;
+
+namespace Glyssen.Utilities
+{
+	struct CustomColor
+	{
+		public static Color BackColor = SystemColors.Control;      // System.Drawing.Color.FromArgb(0, 73, 108)
+		public static Color ForeColor = SystemColors.WindowText;   // System.Drawing.Color.White
+		public static Color LinkColor = SystemColors.HotTrack;     // System.Drawing.Color.FromArgb(51, 153, 255)
+		public static Color Warning = Color.Red;
+		public static Color Highlight1 = Color.DarkBlue;           // Color.Yellow
+		public static Color Highlight2 = Color.MediumBlue;         // Color.LawnGreen
```

Comment:
Yes, and because yellow, lawn green and orange are not close to any default colors.